package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "form-amd",
-  "version": "1.0.3-SNAPSHOT",
+  "version": "1.0.3",
```

Comment:
Are you going to deploy immediately after merging? If not so, it's better to keep the version unchanged.

@luiz